src/main/scala/is/hail/utils/richUtils/RichInt.scala

```diff
@@ -4,4 +4,6 @@ import is.hail.utils.UInt
 
 class RichInt(val i: Int) extends AnyVal {
   def toUInt: UInt = UInt(i)
+
+  def toUIntFromRep: UInt = UInt.applyUnchecked(i)
```

Comment:
I thought we were going to use something like uintFromRep everywhere instead of applyUnchecked (what >> how).